test/src/test/java/org/axonframework/test/saga/AnnotatedSagaTest.java

```diff
@@ -44,6 +44,7 @@
  *
  * @author Allard Buijze
  */
+@Disabled("#3710 Reenable after ParameterResolver fix")
```

Comment:
Should be fixed once the old Saga uses the new UnitOfWork for migration purposes.